server/config/routes.rb

```diff
@@ -5,11 +5,11 @@
   get '/logged_in', to: 'sessions#is_logged_in?'
   
   get '/users/find', to: 'users#show_by_find'
-  resources :users, only: [:create, :show, :index] 
+  get '/company-representative-signup', to: 'users#new'
+  post '/company-representative-signup', to: 'users#create'
```

Comment:
I don't think we should have a separate route for company representatives. I think we need to scrap this and just have the controller handle the differences between usertypes.

Reply:
these two lines are to display the form in app>view>users>new.html.erb, and post the information collected to the database.

Reply:
I still don't get how new is different from create

Reply:
Why do you need to create a user object to get the class attributes? Can't you do it with the class?

Reply:
I'm sorry i got what you mean, I'll continue on ryan's comment